spring-boot-starter-camunda/src/main/java/io/camunda/zeebe/spring/client/properties/ZeebeClientConfigurationProperties.java

```diff
@@ -194,6 +201,24 @@ public void setApplyEnvironmentVariableOverrides(boolean applyEnvironmentVariabl
     this.applyEnvironmentVariableOverrides = applyEnvironmentVariableOverrides;
   }
 
+  public void setScheduledExecutorService(ScheduledExecutorService scheduledExecutorService) {
+    this.scheduledExecutorService = scheduledExecutorService;
+  }
+
+  public void setOwnsJobWorkerExecutor(boolean ownsJobWorkerExecutor) {
+    this.ownsJobWorkerExecutor = ownsJobWorkerExecutor;
+  }
+
+  @Override
+  public ScheduledExecutorService jobWorkerExecutor() {
```

Comment:
```suggestion
  public ScheduledExecutorService getJobWorkerExecutor() {
```

Whats your thought about prefixing this with `get` similar to other getters?

Reply:
@1nb0und Good idea but as it overrides an interface implementation we cant just rename it here. Its coming from: https://github.com/camunda/zeebe/blob/b1a13e14494322053dd6d42f477854357ed5ec3c/clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClientConfiguration.java#L110